framework/source/class/qx/event/handler/Keyboard.js

```diff
@@ -347,7 +347,14 @@ qx.Class.define("qx.event.handler.Keyboard",
         if (type == "keydown")
         {
           // non-printable, backspace or tab
-          if (qx.event.util.Keyboard.isNonPrintableKeyCode(keyCode) || this._emulateKeyPress[keyCode]) {
+          if (qx.event.util.Keyboard.isNonPrintableKeyCode(keyCode) || this._emulateKeyPress[keyCode] ||
+              /*
+               * We need an artificial keypress event for every keydown event.
+               * Newer browsers do not fire keypress for a regular charachter key (e.g when typing 'a')
+               * if it was typed with the CTRL, ALT or META Key pressed during typing, like
+               * doing it when typing the combination CTRL+A
+               */
+              domEvent.ctrlKey || domEvent.altKey || domEvent.metaKey) {
```

Comment:
This concept looks good. Please, however, don't insert comments right in the middle of an expression. That makes the code difficult to read. I'd suggest something like the following, inserted at line 349, and move that comment to before line 349:
```
let isModified = domEvent.ctrlKey || domEvent.altKey || domEvent.metaKey);
```

(And then, of course, change the `if` statement to use `isModified`)